### PR TITLE
Support and docs for deploying with Kustomize.

### PR DESCRIPTION
**Pull request by coryodaniel**
Support and README docs for deploying with kustomize.

I left my fork as the so you can verify the docs. I can submit a commit to the PR w/ your branch before merging.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,33 @@ curl https://raw.githubusercontent.com/estafette/estafette-gke-preemptible-kille
...
+Create a `kustomization.yaml` file:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: default
+commonLabels:
+  app: preemptible-killer
+bases:
+- github.com/coryodaniel/estafette-gke-preemptible-killer//manifests
```
Comment (nvtkaszpir) [inline README.md]: why is there dobule slash in here?

File: manifests/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: preemptible-killer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
```
Comment (nvtkaszpir) [inline manifests/deployment.yaml]: AFAIR prometheus annotations are deprecated, it should be as service + servicemonitor.
I believe it's just a copy/paste from original file.

File: README.md (hunk 1 above)
Comment (coryodaniel) [reply README.md]: it is the go-getter syntax for a [subdirectory](https://github.com/hashicorp/go-getter#subdirectories)

File: manifests/deployment.yaml (hunk 2 above)
Comment (coryodaniel) [reply manifests/deployment.yaml]: They are, this file was rendered w/ helm template and its still the default on [master](https://github.com/estafette/estafette-gke-preemptible-killer/blob/master/chart/estafette-gke-preemptible-killer/values.yaml#L39)

**Pull request merged**